include/range/v3/view/any_view.hpp

```diff
@@ -304,43 +328,53 @@ namespace ranges
 
             template<typename Ref, category Cat>
             struct any_view_interface
+              : any_sentinel_interface
```

Comment:
Ah, a view IS-A sentinel in the OO sense. This is a bit nasty. I would prefer a HAS-A relationship here, but if it would uglify the code, then leave it.